common/djangoapps/student/views.py

```diff
@@ -1236,9 +1236,27 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
                 AUDIT_LOG.warning(u"Login failed - password for user.id: {0} is invalid".format(loggable_id))
             else:
                 AUDIT_LOG.warning(u"Login failed - password for {0} is invalid".format(email))
+
+        # Labster changes
+        # If there was login error, suggest student to login to region server
+        regions = configuration_helpers.get_value('REGIONS', settings.REGIONS)
```

Comment:
`configuration_helpers` is an interesting feature that helps us manage settings directly from Dajngo Admin.
I think we should start using it to simplify changes in settings in the future. @sidorovdmitry what do you think?

![screenshot 2017-09-13 09 12 11](https://user-images.githubusercontent.com/1247491/30362253-d4866b90-9863-11e7-9783-449618e8b5d1.png)
![screenshot 2017-09-13 09 12 28](https://user-images.githubusercontent.com/1247491/30362254-d489b494-9863-11e7-8f59-bc116a285b28.png)



"{\"GB\": {\"name\": \"the United Kingdom\", \"login_url\": \"https://staging-courses-uk.labster.com/dashboard\"}}"

Reply:
@polesye If it will help us, why not? 👍